features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -1623,12 +1623,12 @@
 			<Claim>
 				<ClaimURI>urn:ietf:params:scim:schemas:core:2.0:meta.resourceType</ClaimURI>
 				<DisplayName>Meta - Location</DisplayName>
-				<AttributeID>ref</AttributeID>
+				<AttributeID>userType</AttributeID>
 				<Description>Meta - Location</Description>
 				<Required />
 				<DisplayOrder>1</DisplayOrder>
 				<SupportedByDefault />
-				<MappedLocalClaim>http://wso2.org/claims/resourceType</MappedLocalClaim>
+				<MappedLocalClaim>http://wso2.org/claims/userType</MappedLocalClaim>
```

Comment:
Ideally the wso2 claim should be still the resourceType claim.
Only the mapped attribute needs to be changed. 
I would suggest to introduce a new attribute for the resource type

Reply:
I have changed it back and the attribute is changed to a new attribute.